deel/influenciae/influence/second_order_influence_calculator.py

```diff
@@ -148,14 +148,16 @@ def _compute_pairwise_interactions(self, dataset: tf.data.Dataset):
             A tensor containing the sum of all the interactions of each point we are removing with each other point
             of the group
         """
-        local_ihvp = ExactIHVP(self.model, dataset) if isinstance(self.ihvp_calculator, ExactIHVP) \
-            else ConjugateGradientDescentIHVP(
-            self.model,
-            self.ihvp_calculator.extractor_layer,
-            dataset,
-            self.ihvp_calculator.n_cgd_iters,
-            self.ihvp_calculator.feature_extractor
-        )
+        if isinstance(self.ihvp_calculator, ExactIHVP):
```

Comment:
We should consider refactoring here instead of all those conditionals. I note it here so we do not forget it in future work